Policies/Kubernetes/forbidden-sysctl-profiles/rego/src_test.rego

```diff
@@ -0,0 +1,287 @@
+    count(results) == 0
+}
+
+input_review = {
+    "object": {
+        "metadata": {
+            "name": "nginx"
+        },
+        "spec": {
+            "containers": {
+                "name": "nginx",
+                "image": "nginx",
+            },
+            "securityContext": {
+                "sysctls": [
+                    {
+                        "name": "kernel.shm_rmid_forced",
+                        "value": "0"
+                    },
+                    {
+                        "name": "net.core.somaxconn",
+                        "value": "1024"
+                    }
+                ]
+            }
+        }
+    }
+}
+
+input_init_review = {
+    "object": {
+        "metadata": {
+            "name": "nginx"
+        },
+        "spec": {
+            "containers": {
+                "name": "nginx",
+                "image": "nginx",
+            },
+            "securityContext": {
+                "sysctls": [
+                    {
+                        "name": "kernel.shm_rmid_forced",
+                        "value": "0"
+                    },
+                    {
+                        "name": "net.core.somaxconn",
+                        "value": "1024"
+                    }
+                ]
+            }
+        }
+    }
+}
+
+input_review_empty = {
+    "object": {
+        "metadata": {
+            "name": "nginx"
+        },
+        "spec": {
+            "containers": {
+                "name": "nginx",
+                "image": "nginx",
+            },
+            "securityContext": {
+            }
+        }
+    }
+}
+
+input_review_sysctls_empty = {
+    "object": {
+        "metadata": {
+            "name": "nginx"
+        },
+        "spec": {
+            "containers": {
+                "name": "nginx",
+                "image": "nginx",
+            },
+            "securityContext": {
+                "sysctls": []
+            }
+        }
+    }
+}
+
+input_review_empty_seccontext = {
+    "object": {
+        "metadata": {
+            "name": "nginx"
+        },
+        "spec": {
+            "containers": {
+                "name": "nginx",
+                "image": "nginx",
+            }
+        }
+    }
+}
+
+input_parameters_wildcard = {
+    "forbiddenSysctls": [
+        "*"
+    ]
+}
+
+input_parameters_in_list = {
+    "forbiddenSysctls": [
+        "kernel.shm_rmid_forced",
+        "net.core.somaxconn"
+    ]
+}
+
+input_parameters_one_in_list = {
+    "forbiddenSysctls": [
+        "kernel.shm_rmid_forced"
+    ]
+}
+
+input_parameters_not_in_list = {
+    "forbiddenSysctls": [
+        "kernel.msgmax"
+    ]
+}
+
+input_parameters_in_list_wildcard = {
+    "forbiddenSysctls": [
+        "kernel.*",
+        "net.core*"
+    ]
+}
+
+input_parameters_one_in_list_wildcard = {
+    "forbiddenSysctls": [
+        "kernel.*"
+    ]
+}
+
+input_parameters_centered_wildcard = {
+    "forbiddenSysctls": [
+        "kernel.*forced",
```

Comment:
these are not valid notations for sysctls
> A sysctl pattern ends with a * character, such as kernel.*.
https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/#podsecuritypolicy

Reply:
can we validation if k8s even allows this notation?